data_tests/auth.py

```diff
@@ -1,9 +1,4 @@
-from base64 import b64decode as decode
-
-TOKEN = "3e83667c-c59c-4fda-aa7a-a47346a3cd6a"
-
-EMAIL_FORGOT_PASSWORD = "rabotynet.test.fp@gmail.com"
-FROM_FORGOT_PASSWORD = decode(b'cm9tYV9leHBlcnQ=').decode()
+TOKEN = {'token': "3e83667c-c59c-4fda-aa7a-a47346a3cd6a"}
```

Comment:
what is this token for?

Reply:
this is mock for verification_token for register new user and recover password. Some times it takes about 2 minutes to get email with verification link. To skip this i put token into DB and  use it as verification token